src/stop.sh

```diff
@@ -84,6 +84,11 @@ while [ $# -ge 1 ]; do
 done
 
 if [ $stop_all -eq 1 ]; then
+    CEPH_MNTS=$(findmnt -t ceph -n --raw --output=target)
+    CEPH_FUSE_MNTS=$(findmnt -t fuse.ceph-fuse -n --raw --output=target)
+    [ -n "$CEPH_MNTS" ] && umount -f $CEPH_MNTS
+    [ -n "$CEPH_FUSE_MNTS" ] && umount -f $CEPH_FUSE_MNTS
+
```

Comment:
Should this somehow vet that the filesystem you're unmounting is for the vstart cluster? Forcibly unmounting any cephfs that may be mounted may not be a good idea.

Reply:
Done. I am assuming 'ceph fs' and 'ceph tell' provides from current cluster which is vstart cluster in this case.